std/meta.d

```diff
@@ -1309,6 +1309,67 @@ private template staticMerge(alias cmp, int half, Seq...)
     }
 }
 
+/**
+ * Allows subsetting of alias sequence that are not
+ * necessarily adjacent to each other, i.e. cannot be sliced.
+ *
+ * Params:
+ *  range = a range object preferably an unsigned array to index T
+ *  T = a set of types to be indexed
+ *
+ * Returns: alias sequence of selected types and evaluates to 
+ *  `AliasSeq!(T[range[0]], T[range[1]], ..., T[range[n]])`.
+ *
+ * How:
```

Comment:
Please remove this "How" section. It looks like this in the generated documentation: http://dtest.thecybershadow.net/artifact/website-9d4ba62443d65a5c734b2162e355226919442b70-e1c12bf6546fb4c761d47f664fc38732/web/phobos-prerelease/std_meta.html#.staticSubset